server/protocols/webadmin/webadmin-mailbox/pom.xml

```diff
@@ -259,6 +259,7 @@
                         -javaagent:"${settings.localRepository}"/org/jacoco/org.jacoco.agent/${jacoco-maven-plugin.version}/org.jacoco.agent-${jacoco-maven-plugin.version}-runtime.jar=destfile=${basedir}/target/jacoco.exec
                         -Xms512m -Xmx2048m</argLine>
                     <reuseForks>true</reuseForks>
+                    <forkCount>2</forkCount>
```

Comment:
What's expensive here is running OpenSearch for reindexing tests.

I propose an idea: how about rely on Lucene with RamStore instead of OpenSearch for such tests? We would be able to run **way faster** without harming test relevance.

Thoughts?

Reply:
> What's expensive here is running OpenSearch for reindexing tests.

I will have a look at this.